astropy/coordinates/jparser.py

```diff
@@ -58,7 +58,7 @@ def shorten(name):
         Full object name with J-coords embedded.
     Returns
     -------
-    shortName: str
+    shortName: str.
```

Comment:
This is wrong. Maybe because it has no description?

Reply:
Yeh, the spacing was off and there was no summary.

Reply:
There still is a full stop at the end of this line...

Reply:
Fixed!